bond/bond_test.go

```diff
@@ -127,4 +129,100 @@ var _ = Describe("tuning plugin", func() {
 		})
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	It("verifies the plugin returns correct results for CNI versions 0.3.0, 0.3.1, 0.4.0", func() {
```

Comment:
Could we just combine these 3 tests into 1? 
Seems only difference is the CNI version?

Perhaps create a list containing all the versions and loop through this?
WDYT?

Reply:
The difference is the Results version we expect:
https://github.com/k8snetworkplumbingwg/bond-cni/pull/36/files#diff-663523154c95a1a2ed183e0a982eb928029e2dc37b6504dd680f38f277ef6663R150
https://github.com/k8snetworkplumbingwg/bond-cni/pull/36/files#diff-663523154c95a1a2ed183e0a982eb928029e2dc37b6504dd680f38f277ef6663R182
https://github.com/k8snetworkplumbingwg/bond-cni/pull/36/files#diff-663523154c95a1a2ed183e0a982eb928029e2dc37b6504dd680f38f277ef6663R214

Also the format of the results for 0.2/0.1 is completely different.

Reply:
Okay I understand.